gradle/scripts/defaultBuildProperties.gradle

```diff
@@ -28,7 +28,7 @@ def BuildProperties BUILD_PROPERTIES = new BuildProperties(project)
     .register(new BuildProperty("doNotSignArtifacts", false, "Do not sight Maven artifacts"))
     .register(new BuildProperty("gobblinFlavor", "standard", "Build flavor (see http://gobblin.readthedocs.io/en/latest/developer-guide/GobblinModules/)"))
     .register(new BuildProperty("hadoopVersion", "2.6.0", "Hadoop dependencies version"))
```

Comment:
Can you change hadoopVersion to 2.3.0? This was bumped up to fix the build issue due to hive 1.1.0.